<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>445</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>591</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">backgroud-color:rgb(234,234,234);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>190</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;DIN Light&quot;;</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_log_nome">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;DIN Light&quot;;</string>
    </property>
    <property name="text">
     <string>Logs</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(99,104,175);
</string>
    </property>
    <widget class="QLabel" name="label_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(99, 104, 174);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 64pt &quot;DIN Light&quot;;</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>541</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 68pt &quot;DIN Light&quot;;</string>
    </property>
    <property name="text">
     <string>8-Puzzle</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_log">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_box">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>261</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;DIN Light&quot;;</string>
    </property>
    <item>
     <property name="text">
      <string>Largura</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Profundidade</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_algoritmo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;DIN Light&quot;;</string>
    </property>
    <property name="text">
     <string>Algoritmo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="random">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>101</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;DIN Light&quot;;</string>
    </property>
    <property name="text">
     <string>Embaralhar</string>
    </property>
   </widget>
   <zorder>tableView</zorder>
   <zorder>start</zorder>
   <zorder>label_log_nome</zorder>
   <zorder>Title</zorder>
   <zorder>widget</zorder>
   <zorder>text_log</zorder>
   <zorder>combo_box</zorder>
   <zorder>label_algoritmo</zorder>
   <zorder>random</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
